mirrord/kube/src/api/kubernetes.rs

```diff
@@ -57,6 +57,23 @@ impl KubernetesAPI {
             target,
         }
     }
+
+    pub async fn detect_openshift<P>(&self, progress: &P) -> ()
+    where
+        P: Progress + Send + Sync,
+    {
+        let mut sub_progress = progress.subtask("checking for openshift");
+        let _= Discovery::new(self.client.clone())
+            .run()
+            .await
+            .map(|api_discovery| {
+                if api_discovery.has_group("route.openshift.io") {
+                    sub_progress.success(Some("mirrord is running on openshift, due to default PSP of openshift, mirrord may not be able to create the agent. Please refer to https://mirrord.dev/docs/overview/faq/"));
```

Comment:
Is this a `success`? It kinda reads like a warning?

Reply:
Oh my bad, i didnt push the updated branch